Foundation/NSAttributedString.swift

```diff
@@ -186,11 +186,11 @@ private extension AttributedString {
     }
 }
 
-extension AttributedString: _CFBridgable {
+extension NSAttributedString: _CFBridgable {
```

Comment:
Typical swift style is actually   TypeName: Protocol   which differs from Objective-C.    

I'd rather we take on style on the whole for slcf in a separate pr. I think @mgrebenets is right -- styles is all over the place, and we'd benefit from something consistent.